### PR TITLE
43 Briefing button for overriding group leadership.

### LINKED ISSUE #43: Some way to take leadership of group
Need some way to take leadership of a group. Can go into briefing menu along with FA crosshair etc, or in a different tab. Not sure where else to put it.

Candidate command: https://community.bistudio.com/wiki/selectLeader

### PR DESCRIPTION
**Pull request by costno**
Per issue #43

### CODE REVIEW

File: f/briefing/f_orbatNotes.sqf [hunk 1]
```diff
@@ -150,6 +150,19 @@ if (count _veharray > 0) then {
...
+_orbatText = _orbatText + "<br/>
+OVERRIDE GROUP CONTROL: <br/> 
+ | <execute expression=""
+	if (leader group player != player) then {
+		hintsilent 'Taking control of your group';group player selectLeader player;
+	} else {
+		hintsilent 'You are already the leader of your group';
+	};"">
+Take control of my group</execute> | <br/>
+Please note: this is only to be used if Arma has not updated your group leader accurately after casaulties or group merging.";
```
Comment (NikkoJT) [inline f/briefing/f_orbatNotes.sqf]: > casualties

File: f/briefing/f_orbatNotes.sqf [hunk 2]
```diff
@@ -150,6 +150,19 @@ if (count _veharray > 0) then {
 
 };
 
+// Add button to take control of group when Arma does not update leadership correctly.
+// TO DO find better spot for this and/or don't show until after mission launch.
```
Comment (SamLex) [inline f/briefing/f_orbatNotes.sqf]: Would at the bottom of the `Administration` tab be a better place for this? It generally has more technical info in it (mechanics, etc.) and would seem an okay place to put an action.

**Review by SamLex — DISMISSED** [review]
I think this should probably live elsewhere in the Briefing, but implementation looks good.

File: f/briefing/f_orbatNotes.sqf [hunk 3]
```diff
@@ -153,3 +153,17 @@ if (count _veharray > 0) then {
 // Insert final result into subsection ORBAT of section Notes
 waitUntil {scriptDone f_script_briefing};
 player createDiaryRecord ["diary", ["ORBAT", _orbatText]];
+
+// Add button to take control of group when Arma does not update leadership correctly.
+player createDiaryRecord ["fa3_actions", ["Group Leadership",
```
Comment (SamLex) [inline f/briefing/f_orbatNotes.sqf:158]: As a tiny point, I'd maybe put this elsewhere e.g. in it's own briefing file (or own component entirely). But so small it's fine here

**Review by SamLex — APPROVED** [review]
LGTM

**Pull request merged**